web/src/components/tasks/CheckElevatorRegistrationStatus.tsx

```diff
@@ -38,8 +40,38 @@ export const CheckElevatorRegistrationStatus = (props: Props): ReactElement => {
     } as Municipality;
   });
 
+  useEffect(() => {
+    if (business?.profileData.communityAffairsAddress?.streetAddress1 && formValues.address1 === "") {
+      formValues.address1 = business.profileData.communityAffairsAddress.streetAddress1;
+    }
+    if (business?.profileData.communityAffairsAddress?.streetAddress2 && formValues.address2 === "") {
+      formValues.address2 = business.profileData.communityAffairsAddress.streetAddress2;
```

Comment:
pebble - Shouldn't you use setFormValues here? Isn't it bad practice to set values in a useState variable directly?

Reply:
For both this if statement and the one above it

Reply:
You're totally right, I should not be directly touching these values! Update to use the setter function